.github/workflows/publish.yml

```
@@ -0,0 +1,54 @@
+  DEPLOYER: "0000000000000000000000000000000000000000000000000000000000000000"
+
+jobs:
+  publish:
+    permissions:
+      packages: write
+      contents: read
+
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Checkout"
+        uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)